main.go

```diff
@@ -20,6 +22,14 @@ func main() {
 		log.Fatalf("Failed to parse %s: %v", *yml, err)
 	}
 
+	if *tpls != "" {
+		log.Printf("Parsing templates at path: %s\n", *tpls)
+		templates, err = templates.ParseGlob(filepath.Join(*tpls, "*.html"))
+		if err != nil {
```

Comment:
This is good, but I'd prefer if we didn't overwrite the global variable.

Could we pick between `_templates`and the parsed custom templates here in main and pass that to `CreateHandler`, and plumb it down to the individual handlers?